membership-common/src/main/scala/com/gu/memsub/subsv2/services/CatalogService.scala

```diff
@@ -55,7 +60,7 @@ class CatalogService[M[_]: Monad](productIds: ProductIds, fetchCatalog: M[String
   ): Validation[NonEmptyList[String], P] = {
     many(plans.filter(_.frontendId.contains(frontendId)), name).flatMap {
       case a if a.size == 1 => Validation.s(a.head)
-      case e => Validation.failureNel(s"Too many plans! $e")
+      case e => Validation.failureNel(s"Too many plans! for name $name with frontendId $frontendId $e")
```

Comment:
this was a change in the MDAPI version but not in the -common version, so I've preserved it (same with the below toDisjunction line)